internal/api/token.go

```diff
@@ -491,3 +500,81 @@ func (a *API) updateMFASessionAndClaims(r *http.Request, tx *storage.Connection,
 		User:         user,
 	}, nil
 }
+
+func validateTokenClaims(outputClaims map[string]any) error {
+	schemaLoader := gojsonschema.NewStringLoader(minimumViableTokenSchema)
+	documentLoader := gojsonschema.NewGoLoader(outputClaims)
+	result, err := gojsonschema.Validate(schemaLoader, documentLoader)
```

Comment:
JSON schema should be precompiled in a `var` and only `Validate` should be running here.